cpp/tests/utilities/column_utilities.cu

```diff
@@ -65,8 +65,11 @@ struct column_property_comparator {
 
     EXPECT_EQ(lhs.size(), rhs.size());
 
+    EXPECT_EQ(lhs.null_count(), rhs.null_count());
```

Comment:
Oops, we have a duplicate here...

Reply:
@nvdbaranec shouldn't we only be checking for null_count equality if `check_exact_equality` is true, like this? 

```
    if (lhs.size() > 0 && check_exact_equality) {
      EXPECT_EQ(lhs.nullable(), rhs.nullable());
      EXPECT_EQ(lhs.null_count(), rhs.null_count());
    }
```

Reply:
I think null_count should still be checked even if just doing equivalence. There shouldn't be a situation where two equivalent columns have different null_counts.

Reply:
Thanks Jake. Will just remove the duplicate line here then.